### DISCUSSION TITLE
How to translate Relations in Admin pannel

### DISCUSSION THREAD

**Discussion by kenjilabs** [filamentphp/filament#798] [General]
Hi How to translate relations on the admin panel

![image](https://user-images.githubusercontent.com/2118549/145547819-d4f370e4-3575-43b0-95c8-099055171d0f.png)

I added use CreateRecord\Concerns\Translatable; to the Create Pages but still when I try to edit the relations it shows [object]

**Comment by danharrin** [comment]
Hey,

Translatable relation managers aren't something I've looked into yet. The implementation might be a little tricky as where would you place the language selector?

So, as of today, I'm not gonna say that relation managers have translatable support yet. You'll need to use normal resources for translatable models.

It will be something we can look into in the future though.

**Reply by tombien** [reply]
I think, right side of header ("Edit document") is good place for language selector.

**Comment by loranger** [comment]
I'm in the same case. 
Did you find a workaround, @kenjilabs in order to make it work at least using default language ?

**Reply by danharrin** [reply]
I suggest that you make each relation manager record link to a corresponding resource page, where the data can be translated - instead of using the modals.

**Reply by loranger** [reply]
Thanks, I'll try this ! 👍🏻

**Reply by tombien** [reply]
Hello,
I am working on solution, more info here: https://github.com/laravel-filament/filament/discussions/1514
